pyro/infer/abstract_infer.py

```diff
@@ -289,8 +293,8 @@ def __init__(self, model, posterior, num_samples, keep_sites=None):
         self.num_samples = num_samples
         self.keep_sites = keep_sites
         super(TracePredictive, self).__init__()
-        warnings.warn('This class will be removed in the next release. Use the '
-                      '`pyro.infer.Predictive` class instead.', DeprecationWarning)
+        warnings.warn('This class is deprecated and will be removed in the next release. '
```

Comment:
Maybe we should say "in a future release" rather than "in the next" (here and elsewhere)?